contrib/packaging/rpm/Makefile

```diff
@@ -1,18 +1,17 @@
 include ../../../Makefile.defs
 
-BUILDDIR := "$(CURDIR)/cilium-$(VERSION)"
+CWD := $(CURDIR)
+VERSION := $(shell echo $(VERSION)|cut -c 1-5 -)
```

Comment:
Does that work for `0.1.19-dev`? I think we can keep the `-dev` since it will be removed on ever official release. Defer to @tgraf